enterprise/server/cmd/ci_runner/main.go

```diff
@@ -508,17 +508,6 @@ func run() error {
 		}
 	}
 
-	// Set convenience env vars (best-effort).
-	if *prNumber != 0 {
-		os.Setenv("GIT_PR_NUMBER", fmt.Sprint(*prNumber))
-	}
-	if *commitSHA != "" {
-		os.Setenv("GIT_COMMIT", fmt.Sprint(*commitSHA))
-	}
-	if *pushedBranch != "" {
-		os.Setenv("GIT_BRANCH", fmt.Sprint(*pushedBranch))
-	}
```

Comment:
FYI these changes haven't been released yet. I moved these into the app since it seemed simpler/cleaner to set all the Git metadata in one place.